_rules/active-focus-shortcut-1e9941.md

```diff
@@ -0,0 +1,270 @@
+---
+id: 1e9941
+name: Keyboard shortcut is only active when component has focus
+rule_type: atomic
+
+description: |
+  This rule checks that if keyboard shortcuts using only letter, number, sign or punctuation keys for a user interface component are implemented are only active when that component has focus
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag21:2.1.4: # Character Key Shortcuts (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+
+authors:
+  - João Vicente
+  - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML document](https://dom.spec.whatwg.org/#concept-document).
+
+## Expectation
+
+For each target element if [keyboard shortcuts](https://www.w3.org/WAI/WCAG21/Understanding/character-key-shortcuts.html#dfn-keyboard-shortcut) using only letter, number, sign or punctuation keys for a [user interface component](https://www.w3.org/WAI/WCAG21/Understanding/character-key-shortcuts.html#dfn-user-interface-component) are implemented by the target's content are only active when that component has focus
+
+**Note:** [keyboard shortcuts](https://www.w3.org/WAI/WCAG21/Understanding/character-key-shortcuts.html#dfn-keyboard-shortcut) implemented by other means (e.g. browser extensions) are not considered
+
+## Assumptions
+
+_There are currently no assumptions_

```

Comment:
There are 3 ways to meet the SC : turn off, remap on on-focus. 

Shouldn't we ensure that, if the keyboard shortcut is activable even when the component has not focus, then it can't be turned off ou remap? 

> - This test assumes no functionality was provided by the website for the user to turn-off the shortcut.
> - This test assumes no functionality was provided by the website for the user to remap the shortcut.

Or that should be a composite rule with those beeing input rules?